docs/genai/development/working-with-rag.ipynb

```diff
@@ -22,29 +14,22 @@
-    "3. **Format Prompt**:The returned chunks of data from the vector store are formatted and inserted into the prompt for the LLM.\n",
-    "4. **Model Inference**:The LLM is invoked with the user's question, relevant chunks of data, and additional information such as conversation history.\n",
+    "Retrieval Augmented Generation (RAG) is a common strategy for extending the knowledge of pre-trained LLMs.<br>\n",
+    "It gives the LLM a corpus of knowledge to draw from when answering questions about topics or information that were not in the original training set.\n",
     "\n",
     "![](_static/rag_architecture.jpg)\n",
     "\n",
-    "### Most important parts of RAG architecture\n",
+    "### Parts of RAG architecture\n",
     "\n",
-    "There are a number of open source and proprietary embeddings models, LLMs, and vector stores.\n",
+    "There are a number of open source and proprietary embeddings models, LLMs, and vector stores.<br>\n",
     "\n",
     "\n",
-    "- **Vector Store**: Used to store the embedded documents in a format that allows for semantic search and retrieval.\n",
-    "    - This demo uses [Milvus](https://milvus.io) &mdash; an open-source vector database designed specifically for similarity search on massive datasets of high-dimensional vectors.\n",
-    "- **Embeddings Model**: Used to embed the documents as well as embedding new user queries for semantic search.\n",
-    "    - The [Hugging Face Massive Text Embedding Benchmark (MTEB) Leaderboard](https://huggingface.co/spaces/mteb/leaderboard) is a great place to start when considering an open source embeddings model.\n",
-    "    - This demo uses either OpenAI's [text-embedding-3-small](https://platform.openai.com/docs/guides/embeddings) or Hugging Face's [all-MiniLM-L6-v2](https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2).\n",
-    "- **LLM**: used for generating responses.\n",
-    "    - The [Hugging Face Open LLM Leaderboard](https://huggingface.co/spaces/open-llm-leaderboard/open_llm_leaderboard) is a great place to start when considering an open source LLM.\n",
-    "    - This demo uses either OpenAI's [gpt-3.5-turbo-0125](https://platform.openai.com/docs/models/gpt-3-5-turbo) or Ollama's [Llama3](https://ollama.com/library/llama3). Additional examples using [Hugging Face](https://huggingface.co) models will be coming in the future.\n",
+    "- **Vector Store**: Used to store the embedded documents in a format that allows for semantic search and retrieval.<br>\n",
+    "    - This demo uses [Milvus](https://milvus.io) &mdash; an open-source vector database designed specifically for similarity search on massive datasets of high-dimensional vectors.<br>\n",
+    "- **Embeddings Model**: Used to embed the documents as well as embedding new user queries for semantic search.<br>\n",
```

Comment:
"- **Embedding Model**: Used to embed the documents, as well as embedding new user queries for semantic search.<br>\n",